syntax = "proto3";

package communication;

message Mission {
  string drone_id = 1;
  string mission_data = 2;
}

message StatusUpdate {
  string drone_id = 1;
  string status_message = 2;
}

service CommunicationService {
  rpc SendMission (Mission) returns (StatusUpdate);
  rpc SendStatusUpdate (StatusUpdate) returns (StatusUpdate);
}
